force-app/main/triggers/classes/HOT_ServiceAppointmentHandler.cls

```diff
@@ -300,6 +300,11 @@ public without sharing class HOT_ServiceAppointmentHandler extends MyTriggers {
                 serviceAppointment.Status == 'Annul' || serviceAppointment.Status == 'Cannot Complete'
             ) {
                 retractedServiceAppointments.add(serviceAppointment);
+                if (serviceAppointment.SchedEndTime >= System.today()) {
+                    HOT_InterpreterOnWorkOrderService.setInterpreterOnWorkOrderFromServiceAppointment(
```

Comment:
Denne metoden er en @future metode, så om det kommer mange endringer samtidig så vil denne tryne. Bør legge til i en liste og kjøre metoden utenfor for-løkken.